### PR TITLE
Graphical Update

### PR DESCRIPTION
**Pull request by int5749**
moved files from doc to images and updated ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -328,141 +331,157 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
...
+2. [landroid_status_de.map](https://github.com/nibi79/worxlandroid/images/landroid_status_de.map)
+
+## Iconset
+
+Several Icons have been created in order to suit requirements for robo-mower. Please feel free to download from this repository and place/extract them into ..\conf\icons\classic
+
+1. [Distance](https://github.com/nibi79/worxlandroid/images/distance.zip)
+2. [Zones](https://github.com/nibi79/worxlandroid/images/zones.zip)
+3. [Refresh](https://github.com/nibi79/worxlandroid/images/refresh.png)
+4. [wr102si](https://github.com/nibi79/worxlandroid/images/wr102si.png)
```
Comment (nibi79) [inline README.md:482]: must be landroid.png and renamed

File: README.md [hunk 2]
```diff
@@ -328,141 +331,157 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
...
-                Switch item=LandroidAllocation2 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation3 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation4 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation5 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation6 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation7 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation8 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
-                Switch item=LandroidAllocation9 mappings=[0="Zone 1", 1="Zone 2", 2="Zone 3", 3="Zone 4"]
+        Frame {
+            Text item=LandroidMacAdress
```
Comment (nibi79) [inline README.md:367]: please remove LandroidMacAdress

File: README.md [hunk 3]
```diff
@@ -328,141 +331,157 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
...
-                Setpoint item=LandroidScheduleSundayDuration minValue=15 maxValue=1425 step=15
-                Switch item=LandroidScheduleSundayEdgecut
-            }
+	Group item=Shaun icon="landroid" {
+        Frame {
+			Switch item=LandroidAction label="Action" mappings=[START="Start"] visibility=[LandroidStatusCode==0, LandroidStatusCode==1]
+			Switch item=LandroidAction label="Action" mappings=[STOP="Stop",HOME="Home"] visibility=[LandroidStatusCode==7]
+			Switch item=LandroidAction label="Action" mappings=[STOP="Stop",HOME="Home"] visibility=[LandroidStatusCode==33]
+			Switch item=LandroidAction label="Action" mappings=[START="Start",HOME="Home"] visibility=[LandroidStatusCode==34]
+			Switch item=LandroidPoll label="Refresh" mappings=[ON="Poll"]
```
Comment (nibi79) [inline README.md:362]: format

File: README.md [hunk 4]
```diff
@@ -328,141 +331,157 @@ Number          LandroidScheduleSundayDuration          "Duration [%d]"
...
+        Frame label="Statistic" {
+            Text item=LandroidTotalTime
+            Text item=LandroidTotalDistance label="Total Distance [%.2f km]"
+            Text item=LandroidTotalBladeTime
+        }
+		Frame label="Orientation"{
+			Text item=LandroidPitch
+			Text item=LandroidRoll
+			Text item=LandroidYaw
+		}
```
Comment (nibi79) [inline README.md:444]: format

File: README.md [hunk 5]
```diff
@@ -245,9 +246,11 @@ Bridge worxlandroid:worxlandroidBridge:MyWorxBridge "MyWorx Bridge" [ webapiUser
 
 ### .items
 ```
+String Shaun							"Shaun [%s]"
```
Comment (nibi79) [inline README.md:249]: format

File: README.md (hunk 1 above)
Comment (int5749) [reply README.md]: renamed and updated

File: README.md (hunk 2 above)
Comment (int5749) [reply README.md:367]: ?? why, is this not right place to add this?

File: README.md (hunk 3 above)
Comment (int5749) [reply README.md:362]: weird stuff as it looks fine in VSC on my end, will try to update manually on Git

File: README.md (hunk 4 above)
Comment (int5749) [reply README.md:444]: same than previous comment

File: README.md (hunk 5 above)
Comment (int5749) [reply README.md:249]: solved

File: README.md (hunk 4 above)
Comment (int5749) [reply README.md]: solved

File: README.md (hunk 3 above)
Comment (int5749) [reply README.md]: solved

File: README.md (hunk 2 above)
Comment (int5749) [reply README.md]: removed => solved

File: README.md (hunk 1 above)
Comment (int5749) [reply README.md]: solved

**Pull request merged**